03-terraform-outputs-across-stacks/stacks/EC2/config.tm.hcl

```diff
@@ -0,0 +1,5 @@
+globals {
+  "dependencies" = [
+    "stacks/vpc",
+  ]
+}
```

Comment:
We should always add a "newline at the end of each file", hence the stop sign here.

Here's how to do that with vscode: https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files

And that's why: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline